### ISSUE TITLE
Add image file detect

### ISSUE THREAD

**Issue by horikx** [joshgillies/node-matrix-bundler-cli#5]
Currently only creates folders and file assets. Would be great if we could import images as image assets

_Workaround_: Batch trigger morph asset

**Comment by joshgillies** [comment]
There's already some smarts around asset type detection, via the following module: https://github.com/joshgillies/node-matrix-file-types

Note: Matrix (at least the last time I looked) only considers a small set of image files as image assets. See: https://github.com/joshgillies/node-matrix-file-types/blob/master/index.js#L7

**Comment by horikx** [comment]
When I tried to upload all of my .jpg files in the tarball generated by matrix-bundler, it went in as file assets. Are you saying that matrix should have created the images automatically? Or the bundler should have auto figured it out.

**Comment by joshgillies** [comment]
Yeah, bundler should have the smarts to work out which asset to create based on the input file. That being said, the implementation we use (https://github.com/joshgillies/node-matrix-file-types) could be failing against your particular use case.

**Comment by joshgillies** [comment]
Hrmmm, after some digging I can confirm there is an edge case with the asset detection right now, and have raised the following issue: https://github.com/joshgillies/node-matrix-file-types/issues/1

I'll keep this issue around until the above is closed off.

Thanks again @horikx for bringing this to my attention! 👍
